.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+
+MIT.md 
```

Comment:
It is a[ requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) that you create a test.md file with some text that should be added to your `.`gitignore` file😉I think you haven't implemented that currently😉😉 kindly fix that🤗
also, we are not required to ignore the `MIT` file as this is the license file for this project😉 I suggest you kindly take it away🤗🤗

<img width="572" alt="test_imp2" src="https://user-images.githubusercontent.com/64643995/218956700-e094209e-b693-4a90-baad-c2bdfd6d8b1e.png">